handbook/product/README.md

```diff
@@ -209,6 +209,25 @@ When writing copy, consider whether additional information is necessary before a
 
 When writing the time in the UI using "am" and "pm" abbreviations, write them **without space** between time and abbreviation, with **no punctuation**, and use **lowercase** letters (e.g. Working hours are 8am to 5pm).
 
+**Writing error messages**
+
+When writing error messages in the UI or CLI, follow these rules:
+- If the solution to the error isn't obvious, write a message with the **error** followed by the **solution**. For example, "No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key."
+- If the solution is obvious when additional info is provided, write a message with the **error** followed by **additional info**. For example, "You don’t have permission to run the script. Only users with the maintainer role and above can run scripts."
+
+**Fleetctl commands with `--hosts` or `--host` flag**
+
+When designing CLI experience for commands that target hosts (e.g. `fleetctl query` or `fleetctl mdm run-command`) that include `--hosts`/`--host` flag, in case if non-existing host is specified, use the following error message: `Error: No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.`  
```

Comment:
Mike had a hard time understanding this sentence.

Reply:
**Minimize surface area**

When designing CLI experience for commands that target hosts (e.g. `fleetctl query` or `fleetctl mdm run-command` when including the `--hosts`or `--host` flag), if a non-existing host is specified, use a single error message such as:

```
Error: No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key.
```

Reply:
^Related to https://github.com/fleetdm/fleet/pull/13487/files#r1313409715